Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/docrepository/adapter/dao/DocumentDaoIntTest.java

```diff
@@ -39,13 +39,11 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 
 //Tests are being ignored due to current pipeline process for PR Validator.
 //These tests all *must* pass for DocumentDAO to be considered correct.
 // When the database gets reflected with th new changes, this @Ignore should be removed.
-@Ignore
```

Comment:
Shouldn't this still be ignored?  Was this class added to the commit by mistake?

Reply:
Yes, it was a mistake, but we can actually remove it now - was me testing if that still worked and forgot to add it back. We needed Pull #1645 to go through first since the database scripts were not applied yet for the PR Validator job, so it was failing validation. Should be fine now, so we could actually leave it as is.

Reply:
I would suggest keeping it ignored.  This test forces users to have installed a DB with the scripts before build.  A lot of our users have restrictions on DB usage (which tiers the DB can reside) so might impact community builds.

Reply:
Ok. Adding it back then.